src/panels/config/zwave/ha-config-zwave.js

```diff
@@ -294,6 +330,18 @@ class HaConfigZwave extends LocalizeMixin(PolymerElement) {
         type: Number,
         value: 0,
       },
+
+      protection: {
+        type: Array,
+        value: function () {
+          return [];
+        },
+      },
+
+      protectionNode: {
```

Comment:
Please prefix anything that is internal (not set by a component rendering this component) with an `_`. So make it `_protectionNode`